evalml/automl/automl_search.py

```diff
@@ -452,14 +452,26 @@ def search(self, X, y, data_checks="auto", show_iteration_plot=True):
 
         run_ensembling = self.ensembling
         if run_ensembling and len(self.allowed_pipelines) == 1:
-            logger.warning("Ensembling was set to True, but the number of unique pipelines is one, so ensembling will not run.")
+            logger.warning("Ensembling is set to True, but the number of unique pipelines is one, so ensembling will not run.")
             run_ensembling = False
 
+        if run_ensembling and self.max_iterations is not None:
+            # Baseline + first batch + each pipeline iteration + 1
+            first_ensembling_iteration = (1 + len(self.allowed_pipelines) + len(self.allowed_pipelines) * self._pipelines_per_batch + 1)
```

Comment:
nit-pick: Can this simplify to `(2 + len(self.allowed_pipelines) + len(self.allowed_pipelines) * self._pipelines_per_batch)`?

Reply:
Haha I kept the two 1's separate so it'd be more clear where the math is coming from! But lmk if it doesn't help #_#